documentation/asciidoc/computers/config_txt/overclocking.adoc

```diff
@@ -101,6 +101,10 @@ WARNING: Setting any overclocking parameters to values other than those used by
 
 | temp_soft_limit
 | *3A+/3B+ only*. CPU speed throttle control. This sets the temperature at which the CPU clock speed throttling system activates. At this temperature, the clock speed is reduced from 1400MHz to 1200MHz.  Defaults to `60`, can be raised to a maximum of `70`, but this may cause instability.
+
+| core_freq_fixed
+| Setting to 1 disables active scaling of the core clock frequency and ensures that any peripherals that use the core clock will maintain a consistent speed. The fixed clock speed is the higher/turbo frequency for the platform in use. Use this in preference to setting specific core_clock frequencies as it provides portability of config files between platforms.
```

Comment:
Is this the same thing as `force_turbo=1` which is mentioned in the initial paragraph of this page?

Reply:
Okay, so does the "or disable dynamic clocking by applying a static scaling governor (powersave or performance) or with `force_turbo=1`." wording near the start of this page need to be updated to mention `core_freq_fixed=1` instead?

Reply:
More of a @pelwell question I think. My understanding is that force_turbo affects the ARM clock as well, whilst this option just locks the GPU core clock and leaves the ARM to be scaled as necessary.

Reply:
Correct - I don't think any further changes are required.